### PR TITLE
Update QA config values to AAT QA

### PR DESCRIPTION
**Pull request by imsickofmaps**
from `*120*550` range to `*120*4216` range

### CODE REVIEW

**Comment by imsickofmaps** [comment]
@gsvr review pls

File: config/config_ussd.optout.prd.json [hunk 1]
```diff
@@ -1,29 +0,0 @@
-{
```
Comment (gsvr) [inline config/config_ussd.optout.prd.json:1]: I'm assuming this moved to ndoh-private.

File: config/config_ussd.servicerating.qa.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
     "env": "qa",
     "metric_store": "ndoh_qa",
     "endpoints": {"sms": {"delivery_class": "sms"}},
-    "channel": "*120*550*10#",
```
Comment (gsvr) [inline config/config_ussd.servicerating.qa.json:6]: It seems like channel was wrong here previously - assume it's correct now.

**Comment by gsvr** [comment]
@imsickofmaps It's not necessary, but do we want to change the channels in the test setup for consistency?  Otherwise looks good.

File: config/config_ussd.optout.prd.json (hunk 1 above)
Comment (imsickofmaps) [inline config/config_ussd.optout.prd.json:1]: Yes

File: config/config_ussd.servicerating.qa.json (hunk 2 above)
Comment (imsickofmaps) [inline config/config_ussd.servicerating.qa.json:6]: Yes

**Comment by imsickofmaps** [comment]
:shipit:

**Pull request merged**